### PR TITLE
feat: Implement animated Pie scroll wheel

### PR DESCRIPTION
**Pull request by DesmondTambe**
## Description

- Implemented animated Pie scroll wheel (Circular slider widgets)
- Different budget categories
- Latest budget activities

Challenges:

- Adding multi-slider paints on the circular arc

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

File: lib/core/presentation/theming/themes/theme.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
```
Comment (lkmandy) [inline lib/core/presentation/theming/themes/theme.dart:2]: This import isn't used within this file, please remove it

File: lib/core/presentation/theming/themes/light_theme.dart [hunk 2]
```diff
@@ -63,7 +73,7 @@ ThemeData whiteTheme(BuildContext context, BoxConstraints cts) {
       ),
       titleSmall: style.titleSmall?.copyWith(
         fontSize: cts.maxWidth * 0.04,
-        color: _AppColors.onSurfaceW,
+        color: _AppColors.greyLight,
```
Comment (lkmandy) [inline lib/core/presentation/theming/themes/light_theme.dart:76]: - The greyLight color code doesn't match the two flavors on the design `#CACACA` and `#E0E0E0`
- `lightGrey` would make a better name

Comment (lkmandy) [inline lib/core/presentation/theming/themes/light_theme.dart:76]: - The app background color should be some shade of white `#F7F7F7` not the current greyish color
- The currency is missing the dollar sign as pointed out in the image
-  The like icon should be right-aligned not centered
- The New Category button should be black, not blue
- Is the pie chart currently able to represent different colors for different categories?

pr-rv

**Review by lkmandy — CHANGES_REQUESTED** [review]
Nice work @DesmondTambe 

- Please check out my ui related feedback to improve on a few more things
- Also there's a typo in your latest commit `fix: adjusted foramtter issue found in slider_utils.dart`  change  `foramtter ` to `formatted `

**Review by fonkamloic — DISMISSED** [review]

**Pull request merged**